src/FNAPlatform/SDL2_FNAPlatform.cs

```diff
@@ -954,53 +954,68 @@ public static void UnregisterGame(Game game)
 				if (evt.type == SDL.SDL_EventType.SDL_KEYDOWN)
 				{
 					Keys key = ToXNAKey(ref evt.key.keysym);
-					if (!Keyboard.keys.Contains(key))
+					if (key == Keys.LeftControl)
+					{
+						Keyboard.LeftControlDown = true;
+					}
+					else if (key == Keys.RightControl)
+					{
+						Keyboard.RightControlDown = true;
+					}
+
+					if (evt.key.repeat > 0)
```

Comment:
Super tedious but if we can invert this to be `if (evt.key.repeat == 0)` it will help shrink the diff of this if block which is otherwise pretty much the same as it was before.